react/Figure/FigureBlock.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import Types from 'prop-types'
+import classNames from 'classnames'
+import Figure from './Figure'
+import styles from './FigureBlock.styl'
+import labelStyles from '../Label/styles.styl'
+
+const labelStyle = labelStyles['c-label']
+
+/**
+ * Shows a `Figure` with a label, useful for important numbers.
+ *
+ * A part from `className` and `label`, takes same properties
+ * as `Figure`.
+ */
+const FigureBlock = ({
+  className,
+  label,
+  total,
+  symbol,
+  coloredPositive,
+  coloredNegative,
+  signed,
+  decimalNumbers = 0,
+  figureClassName,
+  withCurrencySpacing
+}) => (
+  <div className={classNames(styles['FigureBlock'], className)}>
+    <div className={labelStyle}>{label}</div>
```

Comment:
Can't we use the `Label` component directly?

Reply:
No it is a `label` element.

Reply:
Of course :)